roles/neutron-common/templates/etc/neutron/policy.json

```diff
@@ -53,13 +66,13 @@
 
     "network_device": "field:port:device_owner=~^network:",
     "create_port": "",
-    "create_port:device_owner": "not rule:network_device or rule:admin_or_network_owner or rule:context_is_advsvc",
-    "create_port:mac_address": "rule:admin_or_network_owner or rule:context_is_advsvc",
-    "create_port:fixed_ips": "rule:admin_or_network_owner or rule:context_is_advsvc",
-    "create_port:port_security_enabled": "rule:admin_or_network_owner or rule:context_is_advsvc",
+    "create_port:device_owner": "not rule:network_device or rule:context_is_advsvc or rule:admin_or_network_owner",
```

Comment:
This just looks like the diff is because the order of the rules is changed. Was this changed in the community version?